pkg/apis/servicecatalog/types.go

```diff
@@ -365,10 +372,20 @@ type BindingSpec struct {
 	// Immutable.
 	InstanceRef v1.LocalObjectReference
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is a set of the parameters to be
 	// passed to the underlying broker.
+	// The inline YAML/JSON payload to be translated into equivalent
+	// JSON object.
```

Comment:
I think there's an additional constraint here that a specific parameter name must only be provided once in the parameters section - is that accurate?